common/build.gradle

```diff
@@ -9,14 +9,8 @@ plugins {
 
 apply from: '../versioning/version_tasks.gradle'
 
-def desugarCode = false
```

Comment:
We'll need to update call sites in pipeline code to cleanup sugaring where its present.